CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ Issue](https://github.com/cylc/cylc-flow/issues) as the team may not be able to
 consider large changes that appear out of the blue. New contributors should
 add their details to the [Code Contributors](#code-contributors) section of
 this file as part of their first Pull Request, and reviewers are responsible
-for checking this before merging the new branch into *master*. 
+for checking this before merging the new branch into *master*.
```

Comment:
Had a look at the rendered markdown, and both the link above and this line are both correctly displayed. So I think removing the leading space should be OK.

Reply:
That wasn't intended, my editor is configured to clean withe spaces at line ends and I managed to miss that file going in.

In any case,  I don't mind changing back as it is not directly related to the cleanup at hand.